tests/spread/integration/diffutils/task.yaml

```
@@ -0,0 +1,23 @@
+summary: Integration tests for diffutils
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
+  rootfs="$(install-slices diffutils_bins)"
+
+  mkdir "${rootfs}/test"
+  echo "This is a test file" > "${rootfs}/test1.txt"
+  echo "This is another test file" > "${rootfs}/test2.txt"
+
+  # Compare two different files with cmp
+  chroot "${rootfs}/" cmp -b /test1.txt /test2.txt | grep -q '/test1.txt /test2.txt differ: byte 10, line 1'
+
+  # Compare with diff
+  cp "${rootfs}/test1.txt" "${rootfs}/test3.txt"
+  [[ "$(chroot "${rootfs}/" diff /test1.txt /test3.txt)" == "" ]]
```

```suggestion
  test "$(chroot "${rootfs}/" diff /test1.txt /test3.txt)" = ""
```

I'd suggest use explicit `test`.